### PR TITLE
create task schema

### PR DESCRIPTION
**Pull request by nk-gujjar**
I created the database schema for tasks.
tell me any required update or change.

### CODE REVIEW

File: server/models/taskInfoModel.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        type: String,
+        required: true,
+        trim: true,
+    },
+    // date of the task
+    date: {
+        type: Date,
+        required: true,
+    },
+});
```
Comment (PavithranGoud) [inline server/models/taskInfoModel.js:31]: add ngo_id reference to ngos table

File: server/models/taskInfoModel.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+const assignedSchema = new mongoose.Schema({
+    // reference to the Task model
+    task_id: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: 'Task',
+        required: true,
+    },
+    // reference to the array of volunteer assigned for particular task
+    volunteer_ids: [{
```
Comment (PavithranGoud) [inline server/models/taskInfoModel.js:57]: change volunteer_ids to user_id

**Review by PavithranGoud — CHANGES_REQUESTED** [review]
I commented on code regarding changes need to be done

**Comment by nk-gujjar** [comment]
done required changes.

**Pull request merged**